packages/app/src/app/components/Stat/index.tsx

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
-import formatNumber from './format-number';
-import { Property, Number, Stat } from './elements';
-
-function StatComponent({ count, name }) {
-  return (
-    <Stat>
-      <Property>{name}</Property>
-      <Number>{formatNumber(count)}</Number>
-    </Stat>
-  );
+import { Property, Number, Container } from './elements';
+
+interface IStatProps {
```

Comment:
@Saeris I think you should use a normal `type` instead of an `interface`

```suggestion
type StatProps = {
```